README.md

```
@@ -306,30 +107,24 @@ obtain it:
-Before deploying OSAC, complete the following steps to prepare your environment.
+### Pre-Installation Steps
 
-### 1. Initialize Submodules
+#### 1. Initialize Submodules
 
-The OSAC installer relies on external components managed as Git submodules. Before
-running Kustomize, you must pull the manifest files into your local directory:
+The OSAC installer uses Git submodules for version tracking:
 
 ```bash
 $ git submodule update --init --recursive --remote
```

markdownlint-cli2 (0.22.1)

[warning] Dollar signs used before commands without showing output

(MD014, commands-show-output)